ManageEngine/adauditplus/ingest/parser.yml

```
@@ -27,6 +27,8 @@ stages:
     actions:
       - set:
           event.kind: "alert"
+          adauditplus.alert.name: "{{ parse_event.message.cs4 }}"
+          adauditplus.alert.severity: "{{ parse_event.message.cs5 }}"
```

I suggest using `event.severity` instead. It will avoid using a custom field.
```suggestion
          event.severity: "{{ parse_event.message.cs5 | int }}"
```

---

I hesitated to be honest since `event.severity` is used by the CEF field `severity` for all events, but OK. It it ok to keep `adauditplus.alert.name` though ?